Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -50,7 +51,7 @@ type Props = $ReadOnly<{|
    *
    * See http://facebook.github.io/react-native/docs/activityindicator.html#color
    */
-  color?: ?string,
+  color?: ?ColorValue,
```

Comment:
I think this change requires this line to change too. It probably doesn't fail flow because of the $FlowFixMe lower in this file. https://github.com/facebook/react-native/blob/master/Libraries/Components/ProgressBarAndroid/ProgressBarAndroid.android.js#L52

Reply:
Thanks, fixed.